pkg/environment/environment.go

```diff
@@ -179,7 +181,7 @@ func GetEnv() EnvVariables {
 		AuthLocation:                os.Getenv(AuthLocationVarName),
 		IngressClass:                os.Getenv(IngressClass),
 		WatchNamespace:              os.Getenv(WatchNamespaceVarName),
-		UsePrivateIP:                os.Getenv(UsePrivateIPVarName),
+		UsePrivateIP:                usePrivateIP,
```

Comment:
maybe enablePrivateIP? appgw doesn't seem to  support use privateIP only

Reply:
usePrivateIP should mean the same

Reply:
also, here we force all rules to be tried to `usePrivateIP`.
`enablePrivateIP` might confuse users since AGIC doesn't enable/add private to AppGateway. It barely uses it.